pkgs/tools/inputmethods/libinput-gestures/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, libinput, xdotool, makeWrapper }:
+stdenv.mkDerivation rec {
+  pname = "libinput-gestures";
+  version = "2.32";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "bulletmark";
+    repo = "libinput-gestures";
+    rev = version;
+    sha256 = "1by6sabx0s8sd9w5675gc26q7yccxnxxsjg4dqlb6nbs0vcg81s7";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase =
```

Comment:
Assuming you don't have to install manually, move the clean-up to the ```postInstall``` phase.

Reply:
Everything after invoking `libinput-gestures-setup`?